src/python/WMComponent/DBS3Buffer/MySQL/NewAlgo.py

```diff
@@ -11,19 +11,11 @@
 from WMCore.Database.DBFormatter import DBFormatter
 
 class NewAlgo(DBFormatter):
-    """
-    _NewAlgo_
 
-    Check for existence of algo, then add if not exists
-    """
-    existsSQL = """SELECT id FROM dbsbuffer_algo WHERE app_name = :app_name AND
```

Comment:
No, it won't. That's what we have the INSERT IGNORE for. We use INSERT IGNORE everywhere else in MySQL inserts, no idea why it had a special check here. Maybe because this code was ancient.